stable/nginx-ingress/values.yaml

```diff
@@ -354,6 +354,10 @@ defaultBackend:
 rbac:
   create: true
 
+  # If true, create & use Pod Security Policy resources
+  # https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+  pspEnabled: false
```

Comment:
would it make sense to have the ci-test values to true for psp? so that we get it tested?

Reply:
i learned something new today, you can follow this: https://github.com/helm/charts/tree/master/test#providing-custom-test-values

Reply:
I mean, I'm not sure that the cluster used in testing needs to have podsecuritypolicies enabled for it to have effect. I can add it and see what happens.